build.sbt

```diff
@@ -295,7 +295,7 @@ lazy val scioHdfs: Project = Project(
   )
 ).dependsOn(
   scioCore,
-  scioTest % "test",
+  scioTest % "test->test",
```

Comment:
From http://www.scala-sbt.org/0.12.2/docs/Getting-Started/Multi-Project.html:

"A useful declaration is "test->test" which means Test depends on Test. This allows you to put utility code for testing in bar/src/test/scala and then use that code in foo/src/test/scala, for example."